docs/user.md

```diff
@@ -798,7 +802,26 @@ E.g : `{% wide_image http://google.fr/images/image125.png "A beautiful sunrise"
 #### Fancybox
 
 `fancybox` tag is deprecated since Tranquilpeak 1.3. Please use `image` tag with `fancybox` class to generate them. More information here : [Image tag](#image) 
-  
+
+### Custom 404 error page
+
+When a user requests a page that the server cannot find, a standard *404* error page will be displayed. To create a custom 404 page that fits the theme first create a `404.md` file in your Hexo `source` folder. 
+
+Hide post meta, actions and comments using front-matter settings:
+
+``` yaml
+title: Page not found
+hideMeta: true
+hideActions: true
+comments: false
+```
+
+Now you can customize your 404 error page just like any other blog post.
+Finally, you need to tell your server to use `/404.html` (which Hexo generates out of `404.md`) as your default 404 error page. Here are tutorials for the most common web servers:
+
+ - [Apache](https://www.digitalocean.com/community/tutorials/how-to-create-a-custom-404-page-in-apache)
+ - [Nginx](https://www.digitalocean.com/community/tutorials/how-to-configure-nginx-to-use-custom-error-pages-on-ubuntu-14-04)
```

Comment:
May worth to reference Amazon Cloudfront/S3 custom errors too: http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/custom-error-pages.html